tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -170,5 +189,9 @@ rm -rf "${PIP_TMP_DIR}"
 rm -rf "${TMP_LOGDIR}"
 
 echo
-echo "=== Smoke test of tensorboard PIP package PASSED ==="
+if [[ "${TEST_URL_FAILED}" == 0 ]]; then
+  echo "=== Smoke test of tensorboard PIP package PASSED ==="
+else
+  echo "=== Smoke test of tensorboard PIP package FAILED ==="
```

Comment:
Whoa, you want to `exit 1` here, right? (Including your final `echo` on 197, presumably.)